dockerfiles/squid/Dockerfile

```diff
@@ -1,6 +1,6 @@
 FROM alpine:3.8
```

Comment:
```suggestion
FROM alpine:3.11
```

Alpine 3.11 has Squid v4.11-r0 package.

Reply:
I mean Squid v4.10-r0.

Reply:
When I run docker build on this file, I get the following error message.

```
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/community/x86_64/APKINDEX.tar.gz
ERROR: unsatisfiable constraints:
  squid-3.5.27-r4:
    breaks: world[squid=4.11-r0]
The command '/bin/sh -c apk add --no-cache squid=4.11-r0' returned a non-zero code: 1
```

Reply:
Thanks @adityapahuja. I clearly ran my test before saving the changes...way too tired.

Reply:
👍